llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -2359,6 +2359,14 @@ def int_amdgcn_mbcnt_hi :
   DefaultAttrsIntrinsic<[llvm_i32_ty], [llvm_i32_ty, llvm_i32_ty],
             [IntrNoMem]>;
 
+def int_amdgcn_bcnt032_lo :
+  ClangBuiltin<"__builtin_amdgcn_bcnt032_lo">,
```

Comment:
Is "bcnt032_lo" the name we want to use?  For comparison:

```
  ClangBuiltin<"__builtin_amdgcn_sad_u8">,
  ClangBuiltin<"__builtin_amdgcn_msad_u8">,
```

follows the mnemonic without an initial "v_".

Reply:
@LU-JOHN I removed the 0, which I feel makes them close to the existing `__builtin_amdgcn_mbcnt_lo`.  What do you think?

Re the option, I think most of the other transformations have options, so I feel this one should as well.